validation/td-json-schema-validation.json

```diff
@@ -110,15 +113,25 @@
           ],
           "minItems": 2,
           "contains": {
-            "$ref": "#/definitions/thing-context-td-uri-v1.1"
+            "allOf": [
+              {
+                "$ref": "#/definitions/thing-context-td-uri-v1.1"
+              },
+              {
+                "$ref": "#/definitions/thing-context-td-uri-v1"
+              }
+            ]
```

Comment:
@egekorkan So, without this change, an invalid context definition is actually not being properly detected for some reason, see here: https://www.jsonschemavalidator.net/s/qp7EUNgs

Reply:
I think there is still a problem. The first entry can be any url if the second one is `https://www.w3.org/2022/wot/td/v1.1`
I don't think we want that to happen...

https://www.jsonschemavalidator.net/s/wDzd30PL

Reply:
With the updated schema from this PR, that is actually being picked up by the validator: https://www.jsonschemavalidator.net/s/C6X5uTuI

Reply:
You are right. I used the wrong schema. I think I found another problem, but it gets dirt and might be difficult to fix. One can flip the context URLs and it is still valid.

https://www.jsonschemavalidator.net/s/h7KdeCHp

I am fine if the schema does not catch this issue 

![grafik](https://github.com/user-attachments/assets/4da9f072-86ab-4925-9fe1-a7ebb96a2be5)

Reply:
Hmm, that is a good point,I wasn't sure if that is actually a bug or if a version 1.1 TD could also just add the 1.0 URL to its context :thinking: Maybe the assertion could even be clarified by an erratum, but then again, I am also not sure if the problem is that severe.

Reply:
The links sadly do not work as the tool seems to be offline? Could you paste them to a gist?